docs/data-structures/invert-binary-tree.mdx

```diff
@@ -0,0 +1,214 @@
0Au5fyAuFPx8FQC6WilylIwYnfAM4GRb8kJWngBqNzrhmTLsB+IfUh+Y3+puEInYv8RlPHfTpFibbIz9pcNscw4NNgX7BK1HpvsDHKWK2xgzdl07umeIdqPz/amcP8j0xtlvZrroYlyg+ylS3FIrOm2S2nvs99H0Vj+MLyenR1+f03Raodaj5eco01svCMKq66nmGkePGr4LVOvaEbXGoJp1E0cZkVm507TbczQ2KuW/nfW2N7qRLr/NiRH8l3pp9/wF02FUrRfay2C8X5ne6bDyIX2YYnU3S+EOlNaO7s6dHpw+zX+pNHAXyJdws/xXj2Fll/XdMPnHHxcpjRitbagDwT4ZPctC46JgTHYztM9ncLRY0Eb7v8DbqMDy69QIvYDT3h3fjABA/vv2YttyPFM/7K0FO9zA394hmbLq/bNAGfrjwdrNJr3BEdteNGqwzd2t65izurL1QZxZPwvNjx+Og5ccf3jvL+6rXpt8fDgY9AaHj7ukcce+3JEiaPvdyfbmYDDY3Ca7ZrD4X/BWSVryfwm2cVaPd9+7RuRn8ZIBkOJ+rggIKsHZhmEZhnzNvXJn9rk1Rmco0J5h7FiGfU1HE3GL/ixoGRBi2I0MFp+MOB0p0Ga6uzWyB0dG3ZQqtO7QmRXWoxm197rpY7csQZutyLDDuDtH1km1iOSmWlvg7rbs4NRi8rxa/Y6Eb7QuRk22dusOnHkixN4UyIvO3ZAf5VH0XTFNfLdfLQ1UvItzGadN/o9/HXK9BKClH8gLn2vzR1Dnmahme+YMIrQf3X3u63mP3ry5d8Ayeo75N7o/Sgx69Yw3SqKCEd25A1SJjoryAJX98bldBmrxOGM0H/0gU622chq/8ayYQFnMVn5Hb0DuD2P7xuyKOBeQ3SRJdkbP8+LFny89I+Q195NsVp17fPDPBp9Xbv8PCM6o3iDkZxJooBy918inj9GJRd0N7jrkkx6JiRcNQLxAIqRjx9l0Ss6P84yhsUkrxLyu8wT3wzA7CzBdI/gCDpTatgTeQijdHB0U4vHBBr3kQQFHirTXzUMGl68ZhIi4Iatdkwm7lp8IlBE14NEo9ZRG3ZITHIyeAVjNRj17ixF1osnmaPnouES7juQOkifnwvsJts0jnxol1+R2gmPkM4PGKiiHF9qk6O+0W7mIIFxRbfoiwjxEG7YnE2Keand2lxcYTjISvlxp+6ao5yN9ZemsaPpa8rmXMckWKaY6VxIX5LrWFmutVvJ30ExOTtmjOAHREpqTDefnHxBFe5Jt2+rohF23Jmq2kiIUZmRZbcgZf9kP1wCWS/Jik6cgN1RZ/knxD92wrH5nb2+vY7UNjqBmT9NPAEb+NNwf4dM/jawiCc7Wmn1g+F5f0+bWhuZ+yOdbyz//hNW7qnXWy+Xy+jr4d71zme5rS6VP+4VrVArDT5ksgmmj2RkbHljev0y1eGI+D/crE8v/yfikNNrqj3swRrnccdBjeP7TbRfG/dj/B7+eetpe2PJ+ilMW1WEhZHhhmOn6nW/vQV0YdWP9EvWuV+VJYQZPcCup1FyftryDvJ378/603cBzshMg3p9mP8Al2uJFiGAf8I/X/9XLWcPLe4i7Nz8PKzN2A/4xmTwLsxPB/vq6g6KA9Jco9guFK5xe5DlRhpc7SNu/pAj2A/6z0h+g+1Hsr6/7CCq6E81+Yd/AZjdYL0WNWsB/H6FRPsZvKsNsQlD+XQz75X5yB5AjXSjoxXd8UYQR5zfly+SaF+c3hUo28q/FdQK4f+IPuYrrRKGA7avfGTHO7nLHSPohwvcYvwHun0UClzfjOgHcP2kQrTyJ60Shgm32tSPm3RskPrnImIl6bpHJ7Cv58Z3YS5qGmtOJwhBTGpryoyOGkeNcJpQ86irWb4DjZKA+DqQ95b29cBidVH3maE+hgMmJuFjtQZi2PHjYVvb3C/DVwOU4MMSJD5Ud3RVDM0FS+r9OujAEn+GGnOgKj+GCDq92O7oounuTcZuwTqpC0l/5BkyHL8f+8r+jfQYw/f3Onn4FdyrpubdfKxP6yeH+fhb0K26YfmA89Dhh7QEO+ivAcPdbtvQzOqSg5b7v7oXoTyihkPc/Id1vwyzot8P0k67Zhx4nXPmG1lyV4Td3mLH3kzD9nb4IhxPlpOmTr5DNw+E3PQv6ZRKWyb1O5xLuSlLv/x7S/idZez9BQmIP3Mc14U4lrX/C2r9f+JIJ/QoLGVoGytM8mdCfRvsLwPTHbsb0mxN3L+ssG5p695KeVehO8vxfSF0PZd9whc9caNz2ayRrQQ+TRj6Qt1fcwHR4Lsgg8rHhMRwOPMudpPusrJBkhqI3bOEbbVqhIBk2vXyZMGFDH4U8JWR65Qsew0NQyNjVSzlx6Qh2oingW/XuzAn8E+8a0eKSVRmteikzdtm7l/i+FcqNX/Zi28DBPYpzHISUZ+Prz835EGo3phfly+TFWimuF5Xv+Aoufpz77yU/LonyY90/m4ILZbqRaXOkqhHlRPeigjPfr7DRSbeyhRCyKCcxfoO9UHoNoRZVNCqvI513JrCFSP6vMBk9QrMVlSwvd5C221mR81Ylu2q7yn6cLlgD3zfRpE/V9yO6cYV1sw+l1Wf5L/cRj/ZuR/BfGRpYLI6EWpsqtpfXOybqNmFJnOa/sn+FeacJpbHN8pTlTR/xktPat5mRmyX7wbeiiNBGn3K5Y5LoNUKF/F6Bw+bK9wxOebbq/mSjT7lc7uspzsmzvg4hyyuV/a/G0g2dC8og3T7oR4C9vtlFqbJDH3L0vTDuB+jCF/QTl9OA81vt0Qa9YHeehXzE+RhC+9twbDn4Z/gVUXeXAcb2Td+y+pbjuu3p7/9N/CHS1VWwVXL4/YtvZLVlWGiazqXV7F+artlMW9jnjKsnw33w54nTzuJ0vAjQguc1PG8x2hjFk2V5wQ9BBa94jYanLDbPcx6w3FMW4f5/6zMli16idikAAAAASUVORK5CYII=" width="400" height="175"/>
```

Comment:
is  it necessary to provide the image as a blob here? Can you simply use a image url instead?

Reply:
> is it necessary to provide the image as a blob here? Can you simply use a image url instead?

Yes i think. Because when this is the URL of the image when I tried to copy :)